### PR TITLE
Simple integration of database and catalogue webpage

### PR DESCRIPTION
**Pull request by kwunkiuma**

### CODE REVIEW

**Comment by kwunkiuma** [comment]
image

File: Bookish.Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -26,5 +27,13 @@ public ActionResult Contact()
 
             return View();
         }
+
+        public ActionResult Catalogue()
+        {
+            ViewBag.Message = "Our selection of books.";
```
Comment (FridaTveit) [inline Bookish.Web/Controllers/HomeController.cs:33]: Why do you need to set this here? I would expect this either to be hardcoded in the cshtml file, or a property of the model if it needed to be configurable 🙂

File: Bookish.Web/Models/CatalogueModels.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using Bookish.DataAccess;
+
+namespace Bookish.Web.Models
+{
+    public class CatalogueModels
+    {
+        public IEnumerable<Book> Books = new BookishService().GetBooks();
```
Comment (FridaTveit) [inline Bookish.Web/Models/CatalogueModels.cs:8]: The call to `GetBooks` should happen in the controller 🙂

File: Bookish.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -24,6 +24,7 @@
                     <li>@Html.ActionLink("Home", "Index", "Home")</li>
                     <li>@Html.ActionLink("About", "About", "Home")</li>
                     <li>@Html.ActionLink("Contact", "Contact", "Home")</li>
```
Comment (FridaTveit) [inline Bookish.Web/Views/Shared/_Layout.cshtml:26]: Do you need these other links? 🙂

File: Bookish.Web/Controllers/HomeController.cs [hunk 4]
```diff
@@ -26,5 +27,13 @@ public ActionResult Contact()
 
             return View();
         }
```
Comment (FridaTveit) [inline Bookish.Web/Controllers/HomeController.cs:29]: Please remove any methods that you're not using 🙂 Make sure you set the default page to be one you're using. That can be whatever you prefer; the catalogue, the loans or a separate homepage with some homepage message. It's set in `Startup.cs` with `MapControllerRoute` 🙂

**Review by FridaTveit — COMMENTED** [review]
Nice start! I've left a few comments. Also from the screenshot:
* Please replace application name with an actual name
* Please remove any navbar links you're not using
* Headers don't usually have full stops 🙂

**Pull request closed without merging**